.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,73 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+## I'm submitting a...
+
+<!-- Check one of the following options with "x" -->
+<pre><code>
+[ ] Regression (a behavior that used to work and stopped working in a new release)
+[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
+[ ] Performance issue
+[ ] Feature request
+[ ] Documentation issue or request
+[ ] Support request => https://github.com/single-spa/single-spa-angular/blob/master/CONTRIBUTING.md
+[ ] Other... Please describe:
+</code></pre>
+
+## Current behavior
+
+<!-- Describe how the issue manifests. -->
+
+## Expected behavior
+
+<!-- Describe what the desired behavior would be. -->
+
+## Minimal reproduction of the problem with instructions
+
+<!--
+For bug reports please provide the *STEPS TO REPRODUCE* and if possible a *MINIMAL DEMO* of the problem via
+https://stackblitz.com
+or
+https://codesandbox.io
+
+* Step 1
+* Step 2
+* Step 3
+
+-->
+
+**Note:** Our policy is that issues reported without a reproduction will be closed immediately and then reopened once a reproduction has been provided. Please respect the developers of this project by doing this. We give of our personal time to work on this project and would rather be spending our time fixing or enhancing the library than chasing down badly described or unreproducable issues.
```

Comment:
Honestly this language is a bit more aggressive than I would prefer. I agree that a demonstration of issues is important, but this language might make people feel like they just won't open the issue because they're afraid of looking dumb

Reply:
I can move templates from that PR you've mentioned.
Agreed, let's change a text a little bit but still, we have to encourage people to leave some reproduction, Google teams close all issues immediately if reproduction is not provided, but they al sre-open issue immediately if the author provides it.

Reply:
P.S. this is only from my perspective: I mostly do not look at those issues where reproduction is not provided. In most cases, it's a waste of time. It's like _a kangaroo court_. :slightly_smiling_face: